syncd/syncd.cpp

```diff
@@ -3150,6 +3194,8 @@ void printUsage()
     std::cout << "    -m --portmap"             << std::endl;
     std::cout << "        Specify port map file" << std::endl;
 #endif // SAITHRIFT
+    std::cout << "    -s --syncMode"             << std::endl;
```

Comment:
if we have option in SAI redis to pass syncmode from orchagent, why do you add option in syncd to enable/disable syncd mode?

Reply:
as mentioned those are 2 independent modules, you need to enable sync mode in both of them independently